### PR TITLE
Fix 404 not found

### PR DESCRIPTION
**Pull request by xichengliudui**
**What this PR does / why we need it**:
This link is invalid, I modified it

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE 
```

### CODE REVIEW

**Comment by xichengliudui** [comment]
If I make a mistake, please let me know. Thank you

File: cmd/deepcopy-gen/main.go [hunk 1]
```diff
@@ -62,8 +62,8 @@ func main() {
 
 	// Override defaults.
 	arguments.OutputFileBaseName = "deepcopy_generated"
-	arguments.GoHeaderFilePath = filepath.Join(args.DefaultSourceTree(), "github.com/kubernetes-sigs/kube-batch/hack/boilerplate/boilerplate.go.txt")
-
+	arguments.GoHeaderFilePath = filepath.Join(args.DefaultSourceTree(), "github.com/kubernetes-sigs/kube-batch/blob/master/hack/boilerplate/boilerplate.go.txt")
```
Comment (k82cn) [inline cmd/deepcopy-gen/main.go:65]: we should run it in $GOPATH :)

**Pull request closed without merging**